HelpSource/scdoc.css

```diff
@@ -1,8 +1,13 @@
+*, *::before, *::after {
+    box-sizing: inherit;
+}
+
 html, body {
+    position: relative;
+    box-sizing: border-box;
```

Comment:
border-box makes browsers include padding in the total computed with / height of elements. so when we say `#foo { height: 33px; }` it's guaranteed to be 33px high regardless of padding. very convenient and commonly used in most modern css.

Reply:
Thanks!